install.sh

```diff
@@ -3620,16 +3633,17 @@ setup_hook_templates() {
 
 ############################################################
 # Installs the command line helper tool at
-#   ~/.githooks/bin/githooks and adds a Git alias for it.
+#   $INSTALL_DIR/.githooks/bin/githooks and adds a Git alias for it.
 #
 # Returns:
 #   None
 ############################################################
 install_command_line_tool() {
-    mkdir -p "$HOME/.githooks/bin" &&
-        echo "$CLI_TOOL_CONTENT" >"$HOME/.githooks/bin/githooks" &&
-        chmod +x "$HOME/.githooks/bin/githooks" &&
-        git config --global alias.hooks "!$HOME/.githooks/bin/githooks" &&
+    mkdir -p "$INSTALL_DIR/.githooks/bin" &&
+        echo "$CLI_TOOL_CONTENT" >"$INSTALL_DIR/.githooks/bin/githooks" &&
+        chmod +x "$INSTALL_DIR/.githooks/bin/githooks" &&
+        git config --global alias.hooks "!$INSTALL_DIR/.githooks/bin/githooks" &&
+        git config --global githooks.installDir "$INSTALL_DIR" &&
```

Comment:
Can we set this somewhere else?
It doesn't feel like it belongs in the `install_command_line_tool` function.
OK if you can't find a better place, just doesn't feel like this is the logical place for it.